### ISSUE TITLE
Re-add code of conduct

### ISSUE THREAD

**Issue by scottrigby** [helm/chart-testing#48]
This was accidentally removed in #35 (thanks @nvtkaszpir for noticing). Just adding an issue to track making sure we add it back to `master`.

**Comment by scottrigby** [comment]
Yeah I was wondering the same… should we add it back in this repo though in the meantime, so we have _something_ until the helm org updates that for all helm projects? Is this something we should cover in a governance meeting?

**Comment by unguiculus** [comment]
Probably

**Comment by technosophos** [comment]
I believe we are required to at least enforce the CNCF CoC, though we can add additional things if the CNCF approves. I'll defer to @michelleN who has worked on this part of CNCF.

**Comment by scottrigby** [comment]
Hey @technosophos good idea 👍 

In Keybase I asked about the process for this (updating Helm projects' CoC, for example, to the [Contributor Covenant](https://www.contributor-covenant.org/)). @mattfarina suggested posting to the Helm Maintainers list [per the governance doc](https://github.com/helm/community/blob/master/governance/governance.md#decision-making-at-the-helm-org-level).

So… this is fun… I just quickly looked through all the [Helm org](https://github.com/helm) repos:

- All those that have CoC files at all still point to the [k8s community CoC](https://github.com/kubernetes/community/blob/master/code-of-conduct.md) (many don't have a CoC file)
- The k8s community CoC just points to the [CNCF CoC](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
- The CNCF CoC is adapted from the Contributor Covenant already

My 2¢ is we should add a central Helm CoC file (in the Helm community repo) which points directly to the CNCF CoC, and then add files in each Helm org repo pointing to the central Helm one (same as kubernetes/kubernetes does). I'll post this and let's see what @michelleN and everyone else thinks?

**Comment by michelleN** [comment]
yep that sounds great @scottrigby! +1 from me

**Comment by scottrigby** [comment]
@mattfarina suggested removing one more layer of indirection: each helm org repo can just link directly to CNCF CoC rather than a central Helm one linking there. Sounds good to me!

K8s never extended the CNCF CoC, so we likely won't need to either. But if we ever do need to we can always add that layer of indirection back 😜

Here's a PR, pending Helm project maintainer consensus (of whatever type we need for this): https://github.com/helm/chart-testing/pull/49

**Comment by viglesiasce** [comment]
Linking to the CNCF CoC directly SGTM.

**Comment by scottrigby** [comment]
I added PRs for all helm org repos, except `rudder-federation` (because it's Archived), and the 3 "OBSOLETE" repos: `charts-classic`, `helm-classic`, and `get-helm` (BTW should we archive those as well?).
